### ISSUE TITLE
Support replacing files

### ISSUE THREAD

**Issue by richardszalay** [richardszalay/fastlane-plugin-act#10]
Based on #9 it sounds like it might be a good idea to support patching files into the archives. For an initial proposal:

```ruby
act(
  archive_path: "example/Example.xcarchive",

  replace_files: {
    "ImageInRootOfApp.png" => "/path/to/source/image.png",
    "/ImageInRootOfArchive.png" => "/path/to/source/image.png"
  },
  remove_files: [
    "RedundantAppImage.png"
  ]
)
```

* `replace_files` feels better than `add_files` as it also implies existing files will be deleted, but is it clear enough that it can be used to add new files?
* The in-package paths should honour the same pattern as `plist_file` where the default is .app relative, and `/` prefix is archive-relative

This won't include patching the media catalog, as that is more complex and deserves it's own feature.

**Comment by kimdv** [comment]
Any status on this @richardszalay ?

Can it also support replacing files inside .ipa?

**Comment by richardszalay** [comment]
@kimdv no, but I happily accept PRs with tests 😄

**Comment by kimdv** [comment]
Well i have never done Ruby before, I will try 🤷‍♂️

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by richardszalay** [comment]
If it helps, neither had I when I wrote this

**Comment by richardszalay** [comment]
Before you start, though, I'd recommend taking a look inside that IPA to confirm that the file you want to replace is actually a plain file. 

Files that are part of the image catalog aren't supported by Act as the format isn't documented.

**Comment by kimdv** [comment]
I actually want to replace a whole list. 
I don’t want to change the properties. To much work and maintenance :)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by kimdv** [comment]
So it should be simple !!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by richardszalay** [comment]
Released as v1.2.0, thanks for taking the time to contribute!
